yoyodyne/train.py

```diff
@@ -22,22 +22,29 @@ class Error(Exception):
     pass
 
 
-def _get_logger(experiment: str, model_dir: str, wandb: bool) -> List:
+def _get_logger(experiment: str, model_dir: str, log_wandb: bool) -> List:
     """Creates the logger(s).
 
     Args:
         experiment (str).
         model_dir (str).
-        wandb (bool).
+        log_wandb (bool).
 
     Returns:
         List: logger.
     """
     trainer_logger = [loggers.CSVLogger(model_dir, name=experiment)]
-    if wandb:
+    if log_wandb:
         trainer_logger.append(
             loggers.WandbLogger(project=experiment, log_model="all")
         )
+        # Tells PTL to log best validation acc
+        import wandb
```

Comment:
I'd just move this to the top of the module it's not very heavy.

Reply:
My thinking was that if a user has not installed and logged into wandb, would this break?

Reply:
I'd say just make it required. (I see now that it's not yet listed in the project file, though it is in requirements.txt. I think we should just make it required. If you can install PyTorch you can install this...)